tests/integration/kubernetes/runtimeclass_workloads/k8s-policy-deployment.yaml

```diff
@@ -29,10 +29,8 @@ spec:
         runAsUser: 1000
       containers:
       - name: master
-        image: quay.io/opstree/redis
+        image: quay.io/prometheus/busybox@sha256:dfa54ef35e438b9e71ac5549159074576b6382f95ce1a434088e05fd6b730bc4
         resources:
           requests:
             cpu: 100m
             memory: 100Mi
-        ports:
-        - containerPort: 6379
```

Comment:
`containerPort` is tested elsewhere but ideally I'd try to keep a reference to that here. I think the prometheus/prometheus container lets use us a port.